### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by annaeliana7**
In this feature I: 

- Add header, headline, work, about and contact me sections to desktop version
- Style desktop version in css

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -383,3 +393,124 @@ header {
...
+  #headline p {
+    font-size: 20px;
+  }
+
+  /* ===============================*\
+           Work
+*\=============================== */
+  .portfolio-snapshot {
+    flex-direction: row;
+  }
```
Comment (Santiago220991) [inline style.css:463]: Kindly make sure that your project sticks as much as possible to the Figma template provided. Please increase the margin between the border of your card and the title. :eyes: :eyes: 
Your design
image

Figma Template
image

File: style.css [hunk 2]
```diff
@@ -383,3 +393,124 @@ header {
...
+
+  /* ===============================*\
+           Contact me
+*\=============================== */
+
+  .wrapper {
+    max-width: 800px;
+    margin: 0 auto;
+  }
+}
```
Comment (Santiago220991) [inline style.css]: Please ensure that your project sticks as much as possible to Figma Template provided. Kindly make sure that the submit button in the desktop version is centered. :+1: :smiley: :smiley: Also please make sure that the width of the text and the inputs are the same as provided in the Figma template.

Your design
image


Figma template
image

File: style.css [hunk 3]
```diff
@@ -383,3 +393,124 @@ header {
...
+
+  .about {
+    width: 50%;
+    padding: 0;
+  }
+
+  .language {
+    flex-direction: row;
+    width: 50%;
+  }
```
Comment (Santiago220991) [inline style.css]: I think you did a great job here however there is something to improve. The relation between these two containers is 40% to 60%. Kindly change the width of these containers to make your project stick to the Figma template.  :pray: :pray: 
Your design
image

Figma design
image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @annaeliana7, 
I'm Santiago your reviewer for this PR. :wave: :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
:heavy_check_mark: Good PR summary.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Good commit messages.
:heavy_check_mark: Meaningful feature name branch.
:heavy_check_mark: Followed GitHub flow.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -383,3 +393,133 @@ header {
...
+
+  .about {
+    width: 40%;
+    padding: 0;
+  }
+
+  .language {
+    flex-direction: row;
+    width: 60%;
+  }
```
Comment (HAMMAS-SALEEM) [inline style.css:501]: s


- [ ] As the previous reviewer suggested, kindly make all these three language boxes of same sizes to follow the Figma template 😊

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @annaeliana7 :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Github Flow
✔️ Following some requirements

Good job on implementing some of the requested changes, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @annaeliana7  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

-  Changes Implemented✔️ 



 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**